test/parallel/test-process-startTime.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+const now = Date.now();
+const diffFromNow = now - (process.startTime / 1000);
+
+// Make it a bit flexible
+assert.ok(diffFromNow > 0);
+assert.ok(diffFromNow < 150);
```

Comment:
You can use `common.platformTimeout(150)` if you want something more platform-specific?